layout/_partials/post/post-copyright.swig

```diff
@@ -1,4 +1,4 @@
-{%- set ccIcon = '<i class="fa fa-fw fa-creative-commons"></i>' %}
+{%- set ccIcon = '<i class="fab fa-fw fa-creative-commons"></i>' %}
```

Comment:
just fused about `fa`/`far`/`fab`? a huge update. thanks.

Reply:
I did not find the official documentation for migration from v4.7 to v5. In fact, this PR is based on this gist: https://github.com/estebandelaf/scripts/blob/master/fontawesome4to5.sh

About new class names:
`fas` for `solid`
`far` for `regular`
`fab` for `brand`